modules/minimint.nix

```diff
@@ -0,0 +1,125 @@
+{ config, lib, pkgs, ... }:
```

Comment:
Could we call this file `fedimint.nix`? We're trying to switch to that name going forward. Let's always call it `fedimint` in the nixos configs.